epictrack-web/src/components/shared/__test__/FilterSelect.cy.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import FilterSelect from "../filterSelect/FilterSelect";
+
+describe("FilterSelect", () => {
+  it("renders correctly", () => {
+    const options = [
+      { label: "Option 1", value: "1" },
+      { label: "Option 2", value: "2" },
+    ];
+
+    cy.mount(<FilterSelect options={options} variant={"inline"} />);
+
+    cy.get("input").should("exist");
+  });
+
+  it("can select an option", () => {
+    const options = [
```

Comment:
You can also include tests for FilterSelect by setting isMulti option true. In that you can test the "ClearSelect, Cancel, Apply" ...etc